test/shared/src/main/scala/rapture/test/test.scala

```diff
@@ -119,7 +119,7 @@ trait TestSuite {
     type Return
 
     def dependencies = Set[Test]()
-    def action(): Return
+    def action: () => Return
```

Comment:
Because in Scala 2.12 Eta-expansion of zero-argument method values is now deprecated. See https://issues.scala-lang.org/browse/SI-7187
So I have explicitly translated it to function